### PR TITLE
Reduces Cargo Export Revenue for Plastic Crates

### PR DESCRIPTION
**Pull request by Sirryan2002**
## What Does This PR Do

- Refactors cargo selling to now calculate credits earned from imports based on crate value and not just how many crates were exported
- Nerfed plastic crate export values to only 3 credits per crate
- Buffed secure crates to earn 25 credits per crate due to increased rarity and inability to craft them

## Why It's Good For The Game

- Players are making too much money off of plastic crates, I forgot you could craft them and since the materials to make them are practically infinite once you get your hands on a chem machine, it's an unregulated resource tap for space credits
- This will ensure that the crew cannot circumvent the intentional scarcity of space credits especially to import large amounts of weapons crates without significant collaboration or work

## Testing

- Booted up test server
- Exported all 3 kinds of crates with differing crate values
- Ensured the import revenue matched what I exported
- Ensured there were no runtimes

## Changelog
:cl:
tweak: Plastic crates are only worth 3 credits per crate when exported
tweak: Secure crates have been buff'd to be worth 25 credits per crate when exported
/:cl:

### CODE REVIEW

**Comment by Komrad822** [comment]
Marmite nerf spotted.

**Comment by ItsMarmite** [comment]
image

File: code/modules/shuttle/supply.dm [hunk 1]
```diff
@@ -211,9 +217,8 @@
 		credits_to_deposit += credits_from_intel
 
 	if(crate_count > 0)
-		var/credits_from_crates = crate_count * SSeconomy.credits_per_crate
```
Comment (Contrabang) [inline code/modules/shuttle/supply.dm:214]: credits_per_crate is used nowhere now with this change, if you want to remove it

File: code/modules/shuttle/supply.dm [hunk 2]
```diff
@@ -132,6 +133,11 @@
 			SSeconomy.sold_atoms += ":"
 			if(!length(MA.contents))
 				SSeconomy.sold_atoms += " (empty)"
+			if(istype(MA,/obj/structure/closet/crate))
```
Comment (Henri215) [inline code/modules/shuttle/supply.dm:136]:
```suggestion
			if(istype(MA, /obj/structure/closet/crate))
```

**Comment by ArcticValor** [comment]
well that and me finding a way to infinitely stack crates on one tile was the only way I have made 20k in a round (too bad I never did take a screenshot)
and honestly, this is fair as many people can make WAY too many points (and I mean depending on how efficient you are you could be making about I think 300credits a minute(?) ( this is also saying it is only one person doing it)

File: code/modules/shuttle/supply.dm (hunk 1 above)
Comment (Sirryan2002) [reply code/modules/shuttle/supply.dm:214]: removed

**Review by farie82 — APPROVED** [review]

**Pull request merged**